core/source/org/libreoffice/ide/eclipse/core/wizards/pages/UnoPackageExportPage.java

```diff
@@ -145,7 +145,7 @@ private void loadData() {
             i++;
         }
 
-        if(selected) {
+        if (selected) {
```

Comment:
Is this whole file still needed?

Reply:
Yeah, this file forms the first page of the Export -> 'LibreOffice Package', the Ant Script gets build from the 'ManifestExportPage' which is the second page in the export. Nothing related to Ant Script building was form the first page, so has to be kept intact, second page 'ManifestExportPage' though had a number of changes.